doc/manual/manual.md

```diff
@@ -2648,6 +2650,88 @@ construct compound values:
 * `mir_tuple_value : [MIRValue] -> MIRValue` construct a tuple with the given
   list of values as elements.
 
+### MIR slices
+
+Slices are a unique form of compound type that is currently only used during
+MIR verification. Unlike other forms of compound values, such as arrays, it is
+not possible to directly construct a slice. Instead, one must take a slice of
+an existing reference value that points to the thing being sliced. The
+following commands are used to construct slices:
+
+* `mir_slice_value : MIRValue -> MIRValue`: the SAWScript expression
+  `mir_slice_value base` is equivalent to the Rust expression `base[..]`,
+  i.e., a slice of the entirety of `base`. `base` must be a reference to
+  an array value.
```

Comment:
```suggestion
  i.e., a slice of the entirety of `base`. `base` must be a reference to
  an array (`&[T; N]` or `&mut [T; N]`), not an array itself.
```
More clarification.  You could instead say `&(*base)[..]` above, but that's a bit verbose IMO